lib/APIDigester/ModuleAnalyzerNodes.cpp

```diff
@@ -1941,7 +1941,6 @@ SwiftDeclCollector::constructConformanceNode(ProtocolConformance *Conform) {
   Conform->forEachTypeWitness(
     [&](AssociatedTypeDecl *assoc, Type ty, TypeDecl *typeDecl) -> bool {
```

Comment:
Should lambda type annotations be removed as well since it doesn't return bool anymore? Here and other places as well =]
```suggestion
    [&](AssociatedTypeDecl *assoc, Type ty, TypeDecl *typeDecl) {
```

Reply:
Oops, no idea why that didn't cause a compile error. Thanks!